Engine/source/scene/sceneContainer.cpp

```diff
@@ -1012,6 +1012,11 @@ bool SceneContainer::_castRay( U32 type, const Point3F& start, const Point3F& en
       F32 currStartX = normalStart.x;
 
       AssertFatal(currStartX != normalEnd.x, "This is going to cause problems in SceneContainer::castRay");
+	  if(_isnan(currStartX))
```

Comment:
hmm here is a post about it: http://stackoverflow.com/questions/570669/checking-if-a-double-or-float-is-nan-in-c